recipes-kernel/linux/4.19/defconfigs/xenclient-uivm/defconfig

```diff
@@ -536,6 +538,7 @@ CONFIG_PARTITION_ADVANCED=y
 # CONFIG_AMIGA_PARTITION is not set
 # CONFIG_ATARI_PARTITION is not set
 # CONFIG_MAC_PARTITION is not set
+CONFIG_MSDOS_PARTITION=y
 # CONFIG_MSDOS_PARTITION is not set
```

Comment:
This looks weird.  Looks like 4.14 has `CONFIG_MSDOS_PARTITION=y`, so do we want to drop `# CONFIG_MSDOS_PARTITION is not set`

Reply:
I'm good with dropping it. @jean-edouard if you approve I can have an update to this PR with the change.

Reply:
Sorry, just saw that, please submit a separate PR for this, as I'm about to merge. Thanks

Reply:
Ok. Merge away.